src/ecu/utils/helpers.ts

```
@@ -318,7 +318,36 @@
   if (!rawResponse) return '';
 
-  void log.debug(`[Helper:assemble] Input: ${rawResponse}`); // Use logger
+  void log.debug(`[Helper:assemble] Input: ${rawResponse}`);
+  
+  // Remove prompt and clean whitespace
+  const cleanedResponse = rawResponse.replace(/[>\r\n\s]/g, '').toUpperCase();
+  
+  // For CAN responses starting with 7E8 or 7E9
+  if (cleanedResponse.match(/^7E[89]/)) {
+    void log.debug(`[Helper:assemble] Detected CAN format response`);
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)